js/dcf-slideshow.js

```diff
@@ -107,20 +112,28 @@ class DCFSlideshow {
       let ctrlNextButton = document.createElement('button');
 
       // Add classes to slideshow controls group (Keep in DCF)
-      ctrls.classList.add('dcf-list-bare', 'dcf-btn-group', 'dcf-absolute', 'dcf-pin-right', 'dcf-pin-bottom', 'dcf-z-1');
+      ctrls.classList.add('slide-show-control',
+        'dcf-list-bare',
+        'dcf-btn-group',
+        'dcf-absolute',
+        'dcf-pin-right',
+        'dcf-pin-bottom',
+        'dcf-z-1');
 
       // Add role and aria-label to controls group
-      ctrls.setAttribute('role', 'group');
-      ctrls.setAttribute('aria-label', 'slideshow controls');
+      ctrls.setAttribute('aria-label', `${slideShowName} controls`);
+      ctrls.setAttribute('role', 'list');
 
       ctrlPreviousButton.classList.add('dcf-btn', 'dcf-btn-primary', 'dcf-button-slide', 'dcf-btn-slide-prev');
-      ctrlPreviousButton.setAttribute('aria-label', 'previous');
+      ctrlPreviousButton.setAttribute('aria-label', `${slideShowName} previous`);
 
       ctrlNextButton.classList.add('dcf-btn', 'dcf-btn-primary', 'dcf-btn-slide', 'dcf-btn-slide-next');
-      ctrlNextButton.setAttribute('aria-label', 'next');
+      ctrlNextButton.setAttribute('aria-label', `${slideShowName} next`);
 
-      ctrlPrevious.setAttribute('id', 'previous');
-      ctrlNext.setAttribute('id', 'next');
+      ctrlPrevious.setAttribute('id', uuid.concat('-previous'));
+      ctrlPrevious.classList.add('slide-prev-btn');
```

Comment:
Does this need a class? The button already has a `dcf-btn-slide-prev` class on L127 above.

Reply:
Oh, I see why you added it. How about `dcf-li-slide-prev` instead?